network/containerizer/bridgepolicy.go

```diff
@@ -393,6 +367,25 @@ func (p *BridgePolicy) determineContainerSpaces(
 	return spaces.Values(), nil
 }
 
+// spaceNamesForPrinting return a sorted, comma delimited, string containing
+// the space names for each of the given space ids.
+func (p *BridgePolicy) spaceNamesForPrinting(ids set.Strings) string {
+	if ids.Size() == 0 {
+		return "<none>"
+	}
+	names := set.NewStrings()
+	for _, id := range ids.Values() {
+		if name, ok := p.spaceNames[id]; ok {
+			names.Add("\"" + name + "\"")
```

Comment:
Would `names.Add(fmt.Sprintf("%q", name))` be easier?

Reply:
yes, updated